src/library/scala/collection/immutable/Set.scala

```diff
@@ -71,7 +71,7 @@ object Set extends ImmutableSetFactory[Set] {
     def + (elem: Any): Set[Any] = new Set1(elem)
     def - (elem: Any): Set[Any] = this
     def iterator: Iterator[Any] = Iterator.empty
-    override def foreach[U](f: Any =>  U): Unit = {}
+    override def foreach[U](f: Any => U): Unit = {}
```

Comment:
Great whitespace change, but I'd also change the RHS to `()`, which is most expressive and idiomatic. So long as you're touching a LOC.